common/scala/src/main/scala/whisk/core/connector/MessageConsumer.scala

```diff
@@ -202,7 +206,11 @@ class MessageFeed(description: String,
   private def sendOutstandingMessages(): Unit = {
     val occupancy = outstandingMessages.size
     if (occupancy > 0 && handlerCapacity > 0) {
-      val (topic, partition, offset, bytes) = outstandingMessages.dequeue()
+      // Easiest way with an immutable queue to cleanly dequeue
+      // Head is the first elemeent of the queue, desugared w/ a assignment pattern
```

Comment:
a assignment -> an assignment

Reply:
? Not sure what you meant here.

Reply:
typo: desugared w/ a assignment -> desugared w/ an assignment